benchmark/v8/get-stats.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common.js');
+const v8 = require('v8');
+
+const bench = common.createBenchmark(main, {
+  method: [
+    'getHeapStatistics',
+    'getHeapSpaceStatistics'
+  ],
+  n: [1e6],
+  flags: ['--ignition --turbo', '']
```

Comment:
It seems this is a design that may be improved. Currently, key `flags` in configs has no magic impact, while the same flag in options can not vary between runs.

cc @nodejs/performance, @nodejs/benchmarking, @mscdex